apps/sherbet_service/lib/sherbet.service/contact/communication/method/email.ex

```diff
@@ -4,6 +4,7 @@ defmodule Sherbet.Service.Contact.Communication.Method.Email do
     """
 
     @behaviour Sherbet.Service.Contact.Communication.Method
+    @key Application.get_env(:sherbet_service, :email_key_length, 32)
```

Comment:
This should ideally be moved above the `generate_key` function as it's only utilised by that function so makes sense to group them IMO.